VATSpy.dat

```diff
@@ -6925,39 +6926,37 @@ WBKT|Kudat|6.916670|116.833060||WBFC|0
 WBKW|Tawau|4.261667|117.885833||WBFC|0
 WBSB|Brunei|4.944150|114.928606||WBFC|0
 WIBB|Pakanbaru-Simpangtiga|.457500|101.443333||WIIF|0
+WIBJ|Rengat-Japura|.466670|102.316390||WIIF|0
 WICA|Kertajati|-6.658333|108.156944||WIIF|0
-WICB|Tangerang-Budiarto|-6.233060|106.650000||WIIF|0
 WICC|Bandung-Husein Sastranegara|-6.901944|107.576111||WIIF|0
 WICD|Cirebon-Penggung|-6.756144|108.539672||WIIF|0
 WICM|Tasikmalaya-Cibeureum|-7.333060|108.250000||WIIF|0
-WILL|Bandar Lampung-Radin Inten|-5.266670|105.183060||WIIF|0
 WIDN|Tanjung Pinang-Kijang|.918611|104.527222||WSJC|0
 WIDS|Singkep-Dabo|-.483060|104.583060||WIIF|0
-WIHH|Jakarta-Halimperdana Kusuma|-6.267500|106.886389||WIIF|0
-WIHK|Kalijati|-6.549720|107.666390||WIIF|0
-WIHL|Cilacap-Tunggul Wulung|-7.733060|109.016390||WIIF|0
+WIEE|Padang-MinangKabau|-.875656|100.350592||WIIF|0
+WIGG|Bengkulu-Padangkemiling|-3.883060|102.333060||WIIF|0
+WIHH|Halimperdana Kusuma|-6.267500|106.886389||WIIF|0
 WIII|Jakarta-Soekarno-Hatta|-6.123611|106.661111||WIIF|0
+WIJJ|Jambi-Sultan Taha|-1.633060|103.650000||WIIF|0
+WIJI|Kerinci-Depati Parbo|-2.766670|101.366390||WIIF|0

```

Comment:
Can we swap this one and the one above, please? Just to keep them in alphabetical order.

Reply:
Or actually. I've just resolved conflicts between your branch and master, since there was a very large update to all airports not too long ago. If you can have a quick look at it again just to make sure it all makes sense, that'd be great.

Reply:
I already checked it again, all are in order now.